devel/onetbb/Portfile

```diff
@@ -85,6 +85,51 @@ if {[string match *gcc* ${configure.compiler}] && ${configure.build_arch} in [li
 }
 
 variant tests description {Enable tests} {
+    platform darwin {
```

Comment:
What issue is this fixing?

Reply:
It partly fixes malloc errors due to duplicate libstdc++ libs.
(I won’t fight over this one. It is beneficial, since it provides some improvement while not imposing costs, but it does not solve the problem completely in this case.)

Reply:
This is way too complicated and should be removed.

Reply:
Ok, I will drop it now.